src/main/java/com/faforever/client/game/RatingChangeLabelController.java

```diff
@@ -0,0 +1,52 @@
+package com.faforever.client.game;
+
+import com.faforever.client.fx.Controller;
+import com.faforever.client.i18n.I18n;
+import com.faforever.client.replay.Replay.PlayerStats;
+import com.faforever.client.util.RatingUtil;
+import javafx.css.PseudoClass;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+public class RatingChangeLabelController implements Controller<Node> {
+  private static final PseudoClass POSITIVE = PseudoClass.getPseudoClass("positive");
+  private static final PseudoClass NEGATIVE = PseudoClass.getPseudoClass("negative");
+  public Label ratingChangLabelRoot;
+  private final I18n i18n;
+
+  @Inject
+  public RatingChangeLabelController(I18n i18n) {
+    this.i18n = i18n;
+  }
+
+  @Override
+  public Node getRoot() {
+    return ratingChangLabelRoot;
+  }
+
+  @Override
+  public void initialize() {
+    ratingChangLabelRoot.setVisible(false);
+  }
+
+  public void setRatingChange(PlayerStats playerStats) {
+    if (playerStats.getAfterMean() == null || playerStats.getAfterDeviation() == null) {
+      return;
+    }
+    int newRating = RatingUtil.getRating(playerStats.getAfterMean(), playerStats.getAfterDeviation());
+    int oldRating = RatingUtil.getRating(playerStats.getBeforeMean(), playerStats.getBeforeDeviation());
+
+    int ratingChane = newRating - oldRating;
+    ratingChangLabelRoot.setText(i18n.numberWithSign(ratingChane));
+    ratingChangLabelRoot.pseudoClassStateChanged(ratingChane < 0 ? NEGATIVE : POSITIVE, true);
```

Comment:
`ratingChane` -> `ratingChange`

Reply:
What do u mean?

Reply:
`ratingChane` is misspelled, it should be `ratingChane`.
Also, I just see that `ratingChangLabelRoot` is also wrong, it should be `ratingChangeLabelRoot`

Please make sure that you have the spelling checker enabled in your IDE and make sure you look at its reports ;)